app/src/main/res/layout/onboarding_manual_signin.xml

```diff
@@ -26,37 +26,31 @@
         android:id="@+id/header_text"
```

Comment:
Not actual line.

I am seeing a different icon `avatar_icon` for the dark theme in the figma mocks. I assume they should just be the same icon, but let's get that double checked.

Reply:
Yes, I assumed it was mistake as we don't change icons per theme 👍🏽